apps/web/src/services/iam/components/UserGroupTabWorkspaces.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import { reactive, computed } from 'vue';
+
+import { PHeading, PDataTable } from '@cloudforet/mirinae';
+
+import { i18n } from '@/translations';
+
+const state = reactive({
+    loading: false,
+    fields: computed(() => ([
+        { name: 'user_id', label: 'User ID', sortable: false },
+        { name: 'name', label: 'Name', sortable: false },
+        { name: 'state', label: 'State', sortable: false },
+    ])),
+    data: computed(() => ([{
+        user_id: 'user01@spaceone.org',
+        name: 'User01',
+        state: 'Enabled',
+    }])),
```

Comment:
nitpick: Even for temporary data, it is helpful to mark it separately for future work.